contracts/StandardBounty.sol

```diff
@@ -0,0 +1,319 @@
+pragma solidity ^0.4.18;
+import "./inherited/StandardToken.sol";
+
+/// @title StandardBounties
+/// @dev Used to pay out individuals or groups for task fulfillment
+
```

Comment:
@mbeylin as this code is likely to be audited by others more closely. I think it would be great to start building clear comments and docstrings.  I really like the format here: https://github.com/ConsenSys/PLCRVoting/blob/master/contracts/PLCRVoting.sol#L78

I like how it's clearly split into its helpers, etc.. in this case it could be split into bounty interfaces, fulfilment interfaces, etc.  Very clean, and makes for a much readable contract.

Reply:
issue